src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/FinancialTransactionController.java

```diff
@@ -3,21 +3,27 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+
+import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionDTO;
+import pl.byczazagroda.trackexpensesappbackend.dto.UpdateFinancialTransactionDTO;
 import pl.byczazagroda.trackexpensesappbackend.service.FinancialTransactionService;
 
+import javax.validation.Valid;
 import javax.validation.constraints.Min;
 import javax.validation.constraints.NotNull;
 import java.util.List;
 
 @RestController
 @RequiredArgsConstructor
-@RequestMapping("api/transactions")
+@RequestMapping("api/transactions/")
```

Comment:
Should be @RequestMapping("api/transactions")

Reply:
Corrected